### PR TITLE
m4

### PR DESCRIPTION
**Pull request by congnguyendinh0**

### CODE REVIEW

**Comment by gefei** [comment]
Please resize your images. Are you sure you need photos of, say, 3400 x 4500 pixels?

**Comment by congnguyendinh0** [comment]
resized the images

File: content/ws23/master/m4-gamingforthefuture/Elliot.png
Comment (gefei) [inline content/ws23/master/m4-gamingforthefuture/Elliot.png:1]: 951KB? Please resize this photo

File: content/ws23/master/m4-gamingforthefuture/Kenneth.png
Comment (gefei) [inline content/ws23/master/m4-gamingforthefuture/Kenneth.png:1]: Please resize this photo

**Comment by gefei** [comment]
Please resize your photos. I don't think to have photos of 951 KB or 791 KB

**Comment by congnguyendinh0** [comment]
resized and compressed the pictures

**Pull request merged**